.gitignore

```diff
@@ -9,4 +9,10 @@ node_modules
 artifacts
 cache
 tests
-.env
\ No newline at end of file
+.env
+
+# Ignore local deployment files
+.openzeppelin/unknown-*.json
+
+# Don't ignore deployments made to mumbai
+!.openzeppelin/unknown-80001.json
```

Comment:
Does not seem to recognize mumbai so I have to unignore it here.
https://docs.openzeppelin.com/upgrades-plugins/1.x/network-files#network.json